openstack/nova/templates/console-ingress.yaml

```diff
@@ -14,6 +14,7 @@ metadata:
       rewrite "(?i)/{{ $name }}/(.*)" /$1 break;
       rewrite "(?i)/{{ $name }}$" / break;
       {{- end }}
+      {{- include "utils.snippets.set_openstack_request_id" . | indent 6 }}
```

Comment:
Looks like you added the global id to the console ingress instead of the placement ingress by accident.

Reply:
Indeed, sorry about that.